### ISSUE TITLE
SendGrid is blocking my company from logging in and I can't contact support without doing so. Help!

### ISSUE THREAD

**Issue by daverydan** [sendgrid/docs#4927]
## Actual content

**Link to page**:

**Comment by ognyandim** [comment]
This is both funny and not. Let me explain it. Just look at the screenshot below and have fun.

0. You sent me an email with links to support ticket which require for me to log in the system but since my IP was accidentally removed - there is no way. Simple as that.

**1.The first and most effective way is for you to connect to one of the IPs already on your whitelist and send an email to sendgridtesting@gmail.com with a subject of 55c7f1f2720cad7946a788e565edc323. You can do this using our Web API, or SMTP, but you must send through your SendGrid account. ** 

Classical catch 22. I should send you an email from the not allowed IP address? How can I do it?

I can and I did sent you an email from 
**A** the email with which the account is registered (ognyandim@gmail.com) and with which I have contacted the support and received the support ticket

**B** from an e-mail that sits on a server from the last whitelisted IP address .

I showed you that I can change the password of my account via the procedure in which a link is sent to the registered owner email. **Still this is not a proof enough.** 

**2. If you no longer have access to any of the whitelisted IPs, the second method of verifying ownership would be to place a call to the phone number on file for the account and have you read the above code to me over the phone.** 

Really? I have two numbers in a single cell phone and the phone is with me 24/7 and I do not turn it off at all. No call no missed call either. Even though the guy from the support told me that he called me! Really? No way.

**3. The last and least preferred method would be to make a small charge to the credit card on file for the account (less than $1.00, which would be refunded to you) and have you reply to this ticket with the amount of the charge.**

I use SendGrid as extension of my Azure application and they charge me once a month. So your polite support guy told me - there is no way for option 3. 

As a conclusion I can say that your procedure is not well formulated and is logically flawed. I am sure that you will bust more of your clients as time goes by unless you fix this. 

![image](https://user-images.githubusercontent.com/2851260/52327669-95e43c80-29f5-11e9-960e-7f8d70924811.png)

**Comment by ognyandim** [comment]
And to add one more thing - SendGrid allows domain verification via MX records to configure WebHooks. I have configured MX records. But they do not accept this as proof either. This is strange to me.

**Comment by ognyandim** [comment]
Today I decided to create new account. 
The result was that 30 minutes after I have created the new account and set up DNS info in DNS host suddenly the IP was blocked - "Your IP address is not whitelisted....". I am definitely, positively, without shadow of a doubt that I did not touch any IP whitelist configuration.  

I will sadly have to say that I will move on to another provider. I have just exhausted every option. 

I am amazed by this outcome. I used sendGrid for years without a single glitch - it was a perfect service.
